lib/widgets/search_bar.dart

```diff
@@ -35,17 +35,6 @@ class SearchBar extends StatelessWidget {
               padding: EdgeInsets.only(left: 10.0, right: 10.0),
               child: Row(
                 children: <Widget>[
-                  VerticalDivider(
```

Comment:
You can safely remove this whole Padding widget as well, no need to keep it if it doesn't have any children.

Reply:
👍